docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -3,15 +3,15 @@
 For the quick guidelines for the actual content (descriptions, screenshots), see the Appstream [docs](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html).
 
 The following guidelines are optional, but recommended. We will likely promote apps that follow them in certain places.
-Therefor `must` is used in the following text, but it's not a requirement, it means that apps not following these guidelines will likely not be promoted.
+Therefore "must" is used in the following text, but it's not a requirement, it means that apps not following these guidelines will likely not be promoted.
```

Comment:
" don't cause a highlight - I would like to keep it highlighted with `

Reply:
I'm fine with using ` everywhere, but then we should do it consistently across the document, currently it's half and half